### PR TITLE
feature: change the function name about prometheus registry

### PR DESCRIPTION
**Pull request by fengzixu**
Signed-off-by:  fengzixu <hnustphoenix@gmail.com>

### Ⅰ. Describe what this PR did

1. change function name `GetCustomPrometheusRegistry` to `GetPrometheusRegistry`

### Ⅱ. Does this pull request fix one issue?

https://github.com/alibaba/pouch/pull/2429#issuecomment-436150949

### CODE REVIEW

**Comment by fengzixu** [comment]
@fuweid Please review it.

File: pkg/utils/metrics/metrics.go [hunk 1]
```diff
@@ -81,9 +80,9 @@ func NewLabelTimer(subsystem, name, help string, labels ...string) *prometheus.H
 		}, labels)
 }
 
-// GetCustomPrometheusRegistry create a custom resigtry of Prometheus.
-func GetCustomPrometheusRegistry() *prometheus.Registry {
-	return customPrometheusRegistry
+// GetPrometheusRegistry create a custom resigtry of Prometheus.
```
[the remark this answers is not in the document]
Comment (fengzixu) [reply pkg/utils/metrics/metrics.go]: Actually, I don't know your mean. Do you think that returning the resigry of prometheus is wrong ? Or the comment should be writtern as "GetPrometheusRegistry return a resigtry of Prometheus"

[the remark this answers is not in the document]
Comment (fuweid) [reply pkg/utils/metrics/metrics.go]: sorry. Yes. I was saying that the comment should be changed to `GetPrometheusRegistry return a resigtry of Prometheus`. 👍

[the remark this answers is not in the document]
Comment (fengzixu) [reply pkg/utils/metrics/metrics.go]: OK. I fix it at once.

[the remark this answers is not in the document]
Comment (fuweid) [reply pkg/utils/metrics/metrics.go]: forgot to remove `custom`.

**Comment by fuweid** [comment]
@fengzixu thank you!

**Pull request merged**